### DISCUSSION TITLE
Welcome to rme Discussions!

### DISCUSSION THREAD

**Discussion by d-morrison** [d-morrison/rme#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * [Ask questions you’re wondering about](https://github.com/d-morrison/rme/discussions/categories/q-a).
  * [Share ideas](https://github.com/d-morrison/rme/discussions/categories/ideas).
  * [Engage with other community members](https://github.com/d-morrison/rme/discussions/categories/general).
  * [Report typos or larger issues](https://github.com/d-morrison/rme/issues).
     * If you're not sure whether something is a mistake or not, you can ask first in [Q&A](https://github.com/d-morrison/rme/discussions/categories/q-a).
  * [Welcome others and are open-minded](https://github.com/d-morrison/rme?tab=coc-ov-file#readme). Remember that this is a community we build together 💪.

**Comment by Lidya1888** [comment]
A question from Linear regression
, In Table 2.5: Coefficient Interpretations by Model Structure, is the formula you used for the intercept correct? It seems to me that the one you wrote there is for the beta coefficient for male.

**Reply by d-morrison** [reply]
(Moved to #220; thanks for asking!)
